test/hdfs_test.py

```diff
@@ -272,7 +274,13 @@ def test_with_write_success(self):
 
 
 @attr('minicluster')
-class HdfsTargetTests(MiniClusterTestCase):
+class HdfsTargetTests(MiniClusterTestCase, BaseTargetTestCase):
```

Comment:
I think the nonhdfs travis environment would have failed since we don't install hadoop. But I'll double check to be sure.